tests/phpunit/tests/rest-api/rest-server.php

```diff
@@ -2173,4 +2229,19 @@ protected function helper_make_request_and_return_headers_for_rest_send_refreshe
 
 		return rest_get_server()->sent_headers;
 	}
+
+	/**
+	 * Data provider.
+	 *
+	 * @return array
+	 */
+	public function data_embed_params() {
```

Comment:
```suggestion
	public function data_envelope_params() {
```

I think we try to match the data provider name with the test name.